csharp/ql/lib/change-notes/2023-05-16-ilogger-extension-methods.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Arguments in calls to `ILogger` extension methods have been added to `LogMessageSink`, used by the queries `cs/log-forging`, `cs/cleartext-storage`, and `cs/exposure-of-sensitive-information`.
```

Comment:
Should we avoid the implementation detail about `LogMessageSink` here and instead say something like the following?
```suggestion
* The `cs/log-forging`, `cs/cleartext-storage`, and `cs/exposure-of-sensitive-information` queries now correctly handle unsanitized arguments to `ILogger` extension methods.
```

Reply:
Good idea (I simply copied the text over from a previous change note).